sql/parser/expr.go

```diff
@@ -211,7 +211,7 @@ func (node BytesVal) String() string {
 type IntVal int64
 
 func (node IntVal) String() string {
-	return strconv.FormatInt(int64(node), 10)
+	return strconv.FormatUint(uint64(node), 10)
```

Comment:
As I mentioned in the other comment, `IntVal` is storing an unsigned integer in an int64. I'll add an explanatory comment.